iosxr_iso2vbox.py

```diff
@@ -168,6 +168,27 @@ def verboseprint(*args):
         def verboseprint(*args):
             pass
 
+    def cleanup_vms(name, box_name):
+        '''
+        Cleans up any running box with the passed in name.
+        Unregisters and deletes any box with the passed in box_name
+        '''
+        # Remove any running boxes with the same name
+        vms_list_running = str(run('vboxmanage list runningvms'))
+        if name in vms_list_running:
+            verboseprint("'%s' is running, powering off..." % name)
+            run('VBoxManage controlvm %s poweroff' % name)
+        else:
+            verboseprint("'%s' is not running, nothing to poweroff" % name)
+
+        # Unregister and delete any boxes with the same name
+        vms_list = str(run('vboxmanage list vms'))
+        if name in vms_list:
+            verboseprint("'%s' is registered, unregistering and deleting" % name)
+            run('VBoxManage unregistervm %s --delete' % box_name)
```

Comment:
This is to address #8 correct? If so, make sure your commit message includes a string like "Fixes #8" so that the issue will be auto-linked to your PR and will be automatically be closed once this PR is merged.

https://help.github.com/articles/closing-issues-via-commit-messages/